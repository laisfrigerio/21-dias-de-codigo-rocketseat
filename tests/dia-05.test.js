const { JSDOM } = require('jsdom')
const fs = require('fs')
const path = require('path')

const html = fs.readFileSync(path.resolve(__dirname, '../dia-05/index.html'), 'utf8')

let dom
let container

describe('dia 05', () => {
  beforeEach(() => {
    dom = new JSDOM(html, { runScripts: 'dangerously' })
    container = dom.window.document.body
  })

  test('should match the error 404 page challenge', () => {
    expect(container).toMatchSnapshot();
  });
})