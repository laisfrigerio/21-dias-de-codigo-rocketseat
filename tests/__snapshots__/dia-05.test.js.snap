// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dia 05 should match the error 404 page challenge 1`] = `
<body
  class="main-container"
>
  
  
  <header />
  
  
  <section>
    
    
    <div
      class="error-404"
    >
      
      
      <div>
        4
      </div>
      
      
      <div
        class="face"
      >
        
        
        <span
          class="eye"
        >
          
          
          <span
            class="eyebrow"
          />
          
          
          <span
            class="pupil"
          />
          
        
        </span>
        
        
        <span
          class="eye"
        >
          
          
          <span
            class="eyebrow"
          />
          
          
          <span
            class="pupil"
          />
          
        
        </span>
        
      
      </div>
      
      
      <div>
        4
      </div>
      
    
    </div>
    
    
    <h1>
      Oooops, a página que você está procurando parece que não existe ;(
    </h1>
    
  
  </section>
  
  
  <script
    src="../assets/js/include-html.js"
  />
  
  
  <script
    src="./scripts.js"
  />
  



</body>
`;
